pkg/image/api/dockertypes.go

```diff
@@ -121,3 +121,27 @@ type DockerV1CompatibilityImage struct {
 type DockerV1CompatibilityImageSize struct {
 	Size int64 `json:"size,omitempty"`
 }
+
+// DockerConfigImage stores the image configuration
+type DockerConfigImage struct {
+	DockerImage `json:",inline"`
+	RootFS      *DockerConfigRootFS   `json:"rootfs,omitempty"`
+	History     []DockerConfigHistory `json:"history,omitempty"`
+	OSVersion   string                `json:"os.version,omitempty"`
+	OSFeatures  []string              `json:"os.features,omitempty"`
+}
+
+// DockerConfigHistory stores build commands that were used to create an image
+type DockerConfigHistory struct {
+	Created    unversioned.Time `json:"created"`
+	Author     string           `json:"author,omitempty"`
+	CreatedBy  string           `json:"created_by,omitempty"`
```

Comment:
I see, let's keep it this way then. Also `DockerV1CompatibilityImage` follows the same pattern.